### DISCUSSION TITLE
Weird `move to workspace` behaviour when already on that workspace if `move to workspace` is yes.

### DISCUSSION THREAD

**Discussion by pavel-kolesnikov** [i3/i3#4959] [Q&A] [answered]
Hi community!

I'm using `workspace_auto_back_and_forth yes` and have a rule: 
```
for_window [instance="^app.slack.com_.*"] move to workspace 11:F1
```
which should move the Slack app to the `F1` workspace. (Side note. Slack app is a PWA in Chromium, which sets `window instance id` too late to use it with the `assign` clause, so there's `for_window`). 

The problem:
- when I'm on the `F1` workspace already
- and the Slack window launches
- Slack window appears on a whichever workspace I've been before switching to current `F1`.

I need Slack to always appear on `F1` and I'd like the `workspace_auto_back_and_forth` to stay with me.
How?

**Comment by Airblader** [comment, marked as answer]
`move` supports `--no-auto-back-and-forth`.

```
for_window [instance="^app.slack.com_.*"] move --no-auto-back-and-forth to workspace 11:F1
```

**Reply by pavel-kolesnikov** [reply]
Great, thanks.
I've totally missed the section [6.8. Changing (named) workspaces/moving to workspaces](https://i3wm.org/docs/userguide.html#_changing_named_workspaces_moving_to_workspaces)
